packages/webapp/src/components/Finances/Transaction/ExpandedContent/styles.module.scss

```diff
@@ -38,3 +37,30 @@ button.toDetail {
     margin-left: 8px;
   }
 }
+
+.labourTable {
+  display: flex;
+  flex-flow: column nowrap;
+  justify-content: center;
+  align-items: center;
+  background: #f6fbfa;
+
+  .tabs {
+    margin: 24px 0 24px 0;
+    padding-block: 16px;
```

Comment:
The top margin seems to be around 8px according to Figma, looks closer to the "View" link